### PR TITLE
Updated depencies

### LINKED ISSUE #11: Update napps dependencies
- This napp is using python 3.6
- Dependencies have not been updated causing error with loops:
```
    def setUp(self):
        """Execute steps before each tests."""
        patch('kytos.core.helpers.run_on_thread', lambda x: x).start()
        # pylint: disable=import-outside-toplevel
        from napps.kytos.of_l2ls.main import Main
        self.addCleanup(patch.stopall)
    
>       self.napp = Main(get_controller_mock())

tests/unit/test_main.py:22: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
.tox/py39/src/kytos/kytos/lib/helpers.py:19: in get_controller_mock
    controller.buffers = KytosBuffers()
.tox/py39/src/kytos/kytos/core/buffers.py:148: in __init__
    self.conn = KytosEventBuffer("conn")
.tox/py39/src/kytos/kytos/core/buffers.py:29: in __init__
    self._queue = queue_cls(maxsize=maxsize)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <janus.Queue object at 0x7f8b0066ed10>, maxsize = 0

    def __init__(self, maxsize: int = 0) -> None:
>       self._loop = asyncio.get_running_loop()
E       RuntimeError: no running event loop
```

### PR DESCRIPTION
**Pull request by Alopalao**
Closes #11 
- Dropped Openflow v01
- Fixed tests

### CODE REVIEW

**Review by viniarck — COMMENTED** [review]
@Alopalao, nicely done. Thanks for also taking the opportunity to fix the tests. 

Did you also manage to run it locally? Looks good to me. 

Since you've fixed the tests could you also update `setup.cfg` to have these explicit linters?` linters=pylint,pycodestyle,isort`? I'll link scrutinizer to start running for this repo.

**Comment by Alopalao** [comment]
I ran tests again, sometime passes and other times it shows an error on test `test_handle_packet_in()`:

**Comment by viniarck** [comment]
> 

Right. Feel free to mark these cases with `@pytest.mark.xfail` in the meantime and then if you could create a new issue (enhancement), and then in the future depending on our priority when can try to improve these ones. 

Thanks, @Alopalao

File: requirements/dev.txt [hunk 1]
```diff
@@ -1,68 +1,279 @@
...
-urllib3==1.25.8           # via requests
-virtualenv==20.0.15       # via tox
-watchdog==0.10.2          # via kytos
-wcwidth==0.1.9            # via kytos, prompt-toolkit
-werkzeug==1.0.1           # via flask, kytos
-wrapt==1.11.2             # via astroid
-yala==2.2.0               # via kytos-of-l2ls
+-e git+https://github.com/kytos-ng/kytos.git#egg=kytos
+    # via -r requirements/dev.in
+-e file:///home/alopalao/Desktop/back/of_l2ls
```
[the remark this answers is not in the document]
Comment (Alopalao) [reply requirements/dev.txt]: Changed

**Review by viniarck — APPROVED** [review]
Nicely done, Aldo. Let's ship this.

**Pull request merged**